src/main/java/gameboy/gamepack/racinggame/data/entity/Driver.java

```diff
@@ -0,0 +1,35 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.Random;
+
+public class Driver {
+
+    private final Random random;
+    private static final int MOVE_RANGE = 10;
+    private static final int MOVE_MIN = 4;
+    private Car car;
```

Comment:
```suggestion
    private static final int MOVE_RANGE = 10;
    private static final int MOVE_MIN = 4;

    private final Random random;
    private Car car;
```

관례상 상수와 멤버변수 구분을 위해 개행을 두고
상수를 맨위에 배치하고 그다음 멤버변수를 배치합니다.